File: programmers/[Lv.1]숫자 문자열과 영단어/Solution.java
class Solution {
    public int solution(String s) {
        int answer = 0;
        String[] english = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
        try {
            answer = Integer.parseInt(s);
        } catch(Exception e) {
            for(int i =0; i<english.length;i++) {
                s = s.replaceAll(english[i], Integer.toString(i));
            }
            answer = Integer.parseInt(s);
        }
        return answer;
    }
}

File: programmers/[Lv.1]숫자 문자열과 영단어/Solution.swift
import Foundation

func solution(_ s:String) -> Int {
    var result : String = s
    let english = ["zero", "one", "two", "three","four","five","six","seven", "eight", "nine"]
    for x in 0 ..< english.count {
        result = result.replacingOccurrences(of: english[x], with: String(x))
    }
    return Int(result)!
}